### PR TITLE
Feature/rnp 118

### PR DESCRIPTION
**Pull request by MichielVrins**

### CODE REVIEW

File: android/src/test/java/com/onegini/mobile/sdk/reactnative/AcceptMobileAuthConfirmationUseCaseTests.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    lateinit var mobileAuthOtpRequestHandler: MobileAuthOtpRequestHandler
+
+    @Before
+    fun setup() {
+        mobileAuthOtpRequestHandler = MobileAuthOtpRequestHandler(mobileAuthOtpRequestEventEmitter)
+    }
+
+    @Test
+    fun `When mobile authentication with OTP is not enabled, the promise should reject with that error`() {
```
Comment (PiotrKlis) [inline android/src/test/java/com/onegini/mobile/sdk/reactnative/AcceptMobileAuthConfirmationUseCaseTests.kt:48]: As mentioned in other PR, test title can be slightly improved by adding `Then` after the coma.

File: android/src/test/java/com/onegini/mobile/sdk/reactnative/AcceptMobileAuthConfirmationUseCaseTests.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        disabledMobileAuthentication()
+        AcceptMobileAuthConfirmationUseCase(oneginiSdk, mobileAuthOtpRequestHandler)(promiseMock)
+        verify(promiseMock).reject(MOBILE_AUTH_OTP_IS_DISABLED.code, MOBILE_AUTH_OTP_IS_DISABLED.message)
+    }
+
+    @Test
+    fun `When MA-OTP is enabled and no mobile authentication is in progress, should reject with that error`() {
```
Comment (PiotrKlis) [inline android/src/test/java/com/onegini/mobile/sdk/reactnative/AcceptMobileAuthConfirmationUseCaseTests.kt:55]: We should stick with one pattern - `mobile authentication with OTP` or `MA-OTP`. Personally I'm bigger fan of whole names, though here It's pretty self-exaplanatory so it's your call.

**Review by PiotrKlis — CHANGES_REQUESTED** [review]
Left few minor comments, other than that it's looking good 👍

File: android/src/test/java/com/onegini/mobile/sdk/reactnative/AcceptMobileAuthConfirmationUseCaseTests.kt (hunk 2 above)
Comment (MichielVrins) [reply android/src/test/java/com/onegini/mobile/sdk/reactnative/AcceptMobileAuthConfirmationUseCaseTests.kt:55]: I agree, let's go with full names here, we can cut down on text in other places in the test case to keep it readable.

**Pull request merged**